### PR TITLE
Userclass updatedelete

### PR DESCRIPTION
**Pull request by tjsparks5**
All backend functionality for the user's classes is now complete. The user now has a persistent list of classes being updated in the database. The user should be able to add/update courses and delete courses. The classes to choose from now only load when searching for a class (this cuts down on the load times for ng-repeat). I have also made some html and css adjustments to make the page easier to use.

To test:
(1) Add courses by searching. Make sure the plus turns to a check mark when selecting the class.
(2) Delete courses from your list of classes. Once you search for that same class again it should have a plus next to it again meaning you can add it again.

### CODE REVIEW

File: modules/myclasses/server/routes/myclasses.server.routes.js [hunk 1]
```diff
@@ -17,6 +20,14 @@ module.exports = function(app) {
     .put(myclasses.update)
     .delete(myclasses.delete);
 
+  // Routing for requests with the user's classes
+  app.route('/api/userclasses').put(userclasses.updateclasses);
+  app.route('/api/userclasses').get(userclasses.getclasses);
+  app.route('/api/userclasses/deleteclass').put(userclasses.deleteclass);
```
Comment (Bakushin10) [inline modules/myclasses/server/routes/myclasses.server.routes.js:26]: $http.delete is supposed to be used for delete.

**Review by Bakushin10 — APPROVED** [review]
there are small changes that need to be made in the future, but it looks good for now

File: modules/myclasses/server/routes/myclasses.server.routes.js (hunk 1 above)
Comment (tjsparks5) [reply modules/myclasses/server/routes/myclasses.server.routes.js:26]: In the database I am using findOneAndUpdate() with $pull to remove one of the classes from the user's classes array. I am still performing an update, but just removing an entry in the array. So I think using put is still valid here.

**Review by yotoo2920 — APPROVED** [review]
All looks good, Agreed with Shin Comments.

File: modules/myclasses/server/routes/myclasses.server.routes.js (hunk 1 above)
Comment (Bakushin10) [reply modules/myclasses/server/routes/myclasses.server.routes.js:26]: right, I forgot that we can't simply delete the whole data from the db

**Pull request merged**